services/chat/api.go

```diff
@@ -340,7 +336,8 @@ func (chat *Chat) populateCommunityFields(community *communities.Community) erro
 
 	commChat, exists := community.Chats()[chat.ID]
 	if !exists {
-		return ErrChatNotFound
+		// Skip unknown community chats. They might be channels that were deleted
+		return nil
```

Comment:
Just explaining again that somehow, the Chats we got from the DB contained old channels that were deleted from the Community, so this condition retuned an error.
I'm not sure if it's a bug that the DB contains old channel, but skipping them does the trick anyway.